src/index.html

```diff
@@ -7,8 +7,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="true"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img src="images/logo.png" alt="Logo with MOYO letters" >
+      </a>
+      
+      <nav>
+        <ul class="nav__list">
+          <li><a href="#" class="nav__list-item is-active">Apple</a></li>
```

Comment:
Takie formatowanie jest łatwe do odczytania
```suggestion
          <li>
            <a href="#" class="nav__list-item is-active">
              Apple
            </a>
          </li>
```